projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -49,9 +53,7 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
                     ? (((quotes?.carts)[0]?.entries)[0]?.configurationInfos)[0]
                         ?.configurationValues?.entry
                     : ((quotes?.carts)[0].insuranceQuote?.insuredObjectList
-                        ?.insuredObjects)[0]?.insuredObjectItems,
-                  configurable:
```

Comment:
Removed this `configurable ` template parameter, since quotes are available in the template.